source/CommonJobs/CommonJobs.Application.EvalForm/DTOs/CalificationsEvaluationDto.cs

```diff
@@ -35,7 +35,9 @@ public class CalificationsEvaluationDto
 
         public List<string> Evaluators { get; set; }
 
-        public static CalificationsEvaluationDto Create(EmployeeEvaluation evaluation, List<string> evaluators, string currentPosition, string seniority)
+        public bool IsCompanyEvaluationDone { get; set; }
```

Comment:
No, I thought the same the first time, but _ReadyToDevolution_ will be `true` when the responsible opens it to start the devolution. We don't have a state to identify if the company evaluation has finished.